src/archivist/recorder/repositories/git/index.js

```diff
@@ -0,0 +1,203 @@
+/**
+ * This module is the boundary beyond which the usage of git is abstracted.
+ * Commit SHAs are used as opaque unique IDs.
+ */
+
+import fsApi from 'fs';
+import path from 'path';
+
+import mime from 'mime';
+
+import RepositoryInterface from '../interface.js';
+
+import DataMapper, { COMMIT_MESSAGE_PREFIXES_REGEXP, COMMIT_MESSAGE_PREFIX } from './dataMapper.js';
+import Git from './git.js';
+
+const fs = fsApi.promises;
+const PDF_MIME_TYPE = 'application/pdf';
+
+mime.define({ 'text/markdown': ['md'] }, true); // ensure extension for markdown files is `.md` and not `.markdown`
```

Comment:
This is repeated in https://github.com/ambanum/OpenTermsArchive/pull/875/files#diff-6673716d44e6e04b6852f1537f050fa7ebf0bf5c2c7f55d27a6f2de7f627ddadR7.

Reply:
Yep I knew, currently both are needed. So what do you suggest?